.vscode/launch.json

```diff
@@ -58,13 +75,13 @@
       "cwd": "${workspaceFolder}/core/bentley/",
       "type": "node",
       "request": "launch",
-      "runtimeExecutable": "npm",
+      "runtimeExecutable": "pnpm",
       "runtimeArgs": [
         "test"
       ],
-      "outFiles": [
```

Comment:
I'm guessing outFiles is unneeded? I figure CWD is enough to locate the tests, and constraining the search to the outFiles doesn't save time?

Reply:
it... could? Although from my comment below, responding to Nick, I'm a little skeptical it would have an impact. I'll re add the outFiles, but I need to holistically investigate vitest's performance in general, as a separate PR

Reply:
added back

Reply:
Yeah, skeptical it would make any difference too, but if there is no cost to adding it in, might as well keep it for now. Resolving